### ISSUE TITLE
Feedback wanted!

### ISSUE THREAD

**Issue by rbuchberger** [rbuchberger/jekyll_picture_tag#240]
I want to hear about any of the following:

* How are you deploying your Jekyll site with JPT installed? If you had to jump through some hoops to get it working with a particular service, would you mind sharing? That way we can make it easier for the next person.
* How are you using JPT? There are no analytics, so I really have no idea how people use this plugin, or which features are the most important. I've created a [discussion thread](https://github.com/rbuchberger/jekyll_picture_tag/discussions/210) for this purpose.
* How are the docs? Is it easy to find what you're looking for?
* How are the error messages? Did you run into any that are completely unhelpful?
* What are your pain points? How many WTF moments have you had, and what caused them?
* Has JPT made your life easier?

Thanks!

**Comment by IanCassTwo** [comment]
What happens if I try to use v2 using Ruby 2.5? Debian Stable (v10, Buster) comes with 2.5 and I'd rather not have to install a newer version from source.

**Comment by rbuchberger** [comment]
It'll break; we started using some new features from 2.6. 2.5 is pretty old, and actually officially went EOL yesterday.

You don't have to build from source yourself; handling different versions of programming languagese is such a common problem that there are several good tools out there to handle it. [asdf](https://github.com/asdf-vm/asdf) is the one I use, and can recommend it. [rbenv](https://github.com/rbenv/rbenv) is a ruby-specific alternative.

**Comment by IanCassTwo** [comment]
Yes but then it effectively bypasses Debian package management system. As a rule, I try to avoid doing that. It only causes problems in future. I'll try to find a Debian friendly way of updating to 2.6.

**Comment by ankushg** [comment]
I would love to know if anyone's successfully used JPT in Cloudflare Pages!

**Comment by rbuchberger** [comment]
I just tried a test build, looks like it's similar to netlify. No `avif` or `jp2`, but it can handle jpg,  png, and webp.

```
Libvips known savers: csv, mat, v, vips, ppm, pgm, pbm, pfm, hdr, png, jpg, jpeg, jpe, webp, tif, tiff 
Imagemagick known savers: bzlib, cairo, djvu, fftw, fontconfig, freetype, jbig, jng, jpeg, lcms, lqr, ltdl, lzma, openexr, pangocairo, png, rsvg, tiff, wmf, x, xml, zlib
--

**Comment by holzee** [comment]
I am having a problem using JPT with [Jekyll-Polyglot](https://github.com/untra/polyglot). It re-generates all of the images for every build. JPT seems to be not aware of the generated hash. Any clue?

**Comment by duffn** [comment]
Are you still having issues @sbij? The jpt tester repo builds successfully for me on Netlify with Ubuntu 20.04 and `[webp, original]` using jekyll_picture_tag 2.0.3.

**Comment by sbij** [comment]
@duffn i retried on Netlify (i switched to Cloudflare pages at the time, because of this issue) and now i dont have anymore problems, it works well with my repo on github

**Comment by andreadellacorte** [comment]
Hi @rbuchberger, rather than generating the images locally, could you rely on the https://docs.netlify.com/image-cdn/overview/ ? that's what I've been doing, but I think there may be a workable way to do what you're doing with some SaaS model

**Comment by rbuchberger** [comment]
Hi @andreadellacorte, thanks for the idea! I'm sorry to say that I flat out don't have the time or energy to build a feature like that. I'll happily accept pull requests, however.

**Comment by andreadellacorte** [comment]
I think I built it tonight!! Tested it on me website and it works. 

It's wonky because I'm not the best at liquid filters parameters (it's a bit of a nightmare, I've found), but I think you'd be able to integrate it easily in your more mature set up.

I can make a gist and show you where I'm at, then we can discuss integrating and prs if you think it worthwhile.

**Comment by rbuchberger** [comment]
Sounds great! I'd love to see it.
